docs/source/format/CDataInterface.rst

```diff
@@ -22,38 +22,39 @@ The Arrow C data interface
 Rationale
 =========
 
-Apache Arrow aims to be a universal in-memory format for the representation
-of tabular ("columnar") data, but some projects may face a difficult
-choice between either depending on a fast-evolving dependency such as the
-Arrow C++ library, or having to implement adapters for data interchange
-(for example by reimplementing the Arrow IPC format, which is non-trivial
-and does not allow cross-runtime zero-copy).
+Apache Arrow is designed to be a universal in-memory format for the representation
+of tabular ("columnar") data. However, some projects may face a difficult
+choice between either depending on a fast-evolving project such as the
+Arrow C++ library, or having to reimplement adapters for data interchange,
+which may require significant, redundant development effort.
 
 The Arrow C data interface defines a very small, stable set of C definitions
 that can be easily *copied* in any project's source code and used for columnar
 data interchange in the Arrow format.  For non-C/C++ languages and runtimes,
 it should be almost as easy to translate the C definitions into the
 corresponding C FFI declarations.
```

Comment:
Should the FFI acronym be expanded? Is there a link to a reference/spec?

Reply:
I can indeed expand the acronym. There is no spec, it's just a general concept.